lib/cache/entry.js

```diff
@@ -274,6 +274,10 @@ class CacheEntry {
       const cacheWritePromise = new Promise((resolve, reject) => {
         cacheWriteResolve = resolve
         cacheWriteReject = reject
+      }).catch((err) => {
+        if (body) {
+          body.emit('error', err)
+        }
       })
 
       body = new CachingMinipassPipeline({ events: ['integrity', 'size'] }, new MinipassFlush({
```

Comment:
We don't since that event is always raised on the stream and is not a new event that we need to inform the caching stream of like `integrity` and `size`.